source/developers/plugins/plugin-descriptor.rst

```diff
@@ -40,22 +40,22 @@ a blueprint.  Here's a sample taken from the  ``craftercms-plugin.yaml`` for the
       screenshots:
         - title: Home Page
           description: Screenshot of the homepage
-          url: "https://raw.githubusercontent.com/craftercms/site-plugin-example/master/.crafter/logo.svg"
+          url: "https://raw.githubusercontent.com/craftercms/site-plugin-example/master/static-assets/.crafter/screenshots/default.png"
```

Comment:
The path now is: https://raw.githubusercontent.com/craftercms/site-plugin-example/master/.crafter/screenshots/default.png

Note that we removed `static-assets`